### ISSUE TITLE
Master Detail View has bug while setting background as transparent in narrow window mode

### ISSUE THREAD

**Issue by hupo376787** [Microsoft/WindowsTemplateStudio#1629]
## For new Features:

* **Overview:** What you'd think should be added with scenarios
* **Links:**

## For Bugs:
Master Detail View has bug while setting background as transparent in narrow window mode

### Repro steps

I created a master detail view using latest template, when I resize window to narrow mode(0 - 640), and set controls:MasterDetailsView Background="Transparent", the detail info overlay on the list. See the gif

```
<controls:MasterDetailsView
            Grid.Row="1"
            x:Name="MasterDetailsViewControl"
            Background="Transparent"
            ItemsSource="{x:Bind SampleItems}"
            SelectedItem="{x:Bind Selected, Mode=OneWay}"
            ItemTemplate="{StaticResource ItemTemplate}"
            DetailsTemplate="{StaticResource DetailsTemplate}"
            NoSelectionContentTemplate="{StaticResource NoSelectionContentTemplate}"
            BorderBrush="Transparent" />
```
![1](https://user-images.githubusercontent.com/14960168/34463315-ec3251a2-ee92-11e7-864d-c2c1ae8705ff.gif)

In earlier versions, when there exist "MasterDetailDetailPage", there is no such bug.

### Expected Behavior

When setting controls:MasterDetailsView Background="Transparent", and Click an item, use a page instead, so that the detail info do not overlay on the list.

### Actual Behavior

![1](https://user-images.githubusercontent.com/14960168/34463315-ec3251a2-ee92-11e7-864d-c2c1ae8705ff.gif)

## System

* **VS Version:15.5.2**
* **WTS Wizard Version:1.6.17355.2**
* **WTS Template Version:1.6.17355.2**
* **Windows Build:  16299**

**Comment by crutkas** [comment]
What are your min / target for your project?  Can you share a repro?

**Comment by crutkas** [comment]
@nmetulev have you seen a bug like this in the Toolkit?

**Comment by hupo376787** [comment]
Normally, I set target is 16299, min is 14393. But when I set min is 16299, the bug still exists.
The uwp toolkit is using the same control, so I think it has the same bug when setting background as transparent.
Here is the repo https://github.com/hupo376787/WindowsTemplateStudio---Master-Detail-View-bug.git

**Comment by mvegaca** [comment]
I've tried to set up a different background in Master-Detail page and occurs the same problem. It looks an issue in UWP Community Toolkit, I'll add a new PR with a workaround and open a new issue on UWP Community toolkit repository.

**Comment by hupo376787** [comment]
@mvegaca Thanks for your attention, I think I should use earlier version instead currently.

**Comment by mvegaca** [comment]
I've created a [new Issue](https://github.com/Microsoft/UWPCommunityToolkit/issues/1729) on UWP Community Toolkit.
Added a new [pull request](https://github.com/Microsoft/WindowsTemplateStudio/pull/1630) setting the background to MasterDetail Control.

The bug is only reproduced if a developer manually set the MasterDetail background to Transparent. It's not reproducible when the app is generated out of the box.
We will be waiting for news about that of UWP Toolkit team.

**Comment by mrlacey** [comment]
This is not a bug in anything WTS is doing but is a bug in the community control.
The proposed change does not address the issue as originally reported. Changing the background to transparent still produces the issue.
If this is not a result of something that's changed in WTS templates it should be treated as an external issue and addressed there. 

Can we roll back to an older version of the toolkit that does not have this control?
If, as I suspect, we can't then this may just need to be logged as a known external issue.

**Comment by hupo376787** [comment]
I think we can leave it to the toolkit team to fix, WTS just need to wait for the update. Haha~

**Comment by nmetulev** [comment]
Yeah, this is a known issue (https://github.com/Microsoft/UWPCommunityToolkit/issues/1117), being worked on currently

**Comment by crutkas** [comment]
@mvegaca We shouldn't add in a workaround in our code base, we should add it to the known issue list and in the issue here log the work around.

**Comment by crutkas** [comment]
@mvegaca if you can figure out the solution for the control, i would fix it inside the toolkit with a PR versus altering our code.  That way it ripples to the entire community.

**Comment by hupo376787** [comment]
@nmetulev I saw this bug was discussed for months, but no update until now. Is this situation rarely meet?

**Comment by mvegaca** [comment]
I've not spend my time investigating the bug in the source code of the control, I'll keep following the [Issue](https://github.com/Microsoft/UWPCommunityToolkit/issues/1117) and I could collaborate if needed.

**Comment by nmetulev** [comment]
@skendrot as he has been working on this issue

**Comment by crutkas** [comment]
awesome!  @skendrot rock it!

**Comment by mrlacey** [comment]
Moving to the backlog as an external issue. We'll pick this up when the toolkit has the fix.
